sc_rf_service/sc_rf_service.py

```diff
@@ -47,18 +55,45 @@ class CryomodulePV(PVGroup):
     cryoSumLatchB = pvproperty(value=0, name=":CRYOSUMB_LTCH",
                                dtype=ChannelType.ENUM,
                                enum_strings=("OK", "Faulted"))
-    beamVacuumLatchA = pvproperty(value=0, name=":BMLNVACA_LTCH",
-                                  dtype=ChannelType.ENUM,
-                                  enum_strings=("OK", "Faulted"))
-    beamVacuumLatchB = pvproperty(value=0, name=":BMLNVACB_LTCH",
-                                  dtype=ChannelType.ENUM,
-                                  enum_strings=("OK", "Faulted"))
-    couplerVacuumLatchA = pvproperty(value=0, name=":CPLRVACA_LTCH",
-                                     dtype=ChannelType.ENUM,
-                                     enum_strings=("OK", "Faulted"))
-    couplerVacuumLatchB = pvproperty(value=0, name=":CPLRVACB_LTCH",
-                                     dtype=ChannelType.ENUM,
-                                     enum_strings=("OK", "Faulted"))
+
+
+class RackPV_HWI(PVGroup):
+    hwi = pvproperty(value=0.0, name=":HWINITSUM", dtype=ChannelType.ENUM,
+                     enum_strings=("Ok", "HW Init running", "LLRF chassis problem"))
+
+    def __init__(self, device_name, change_callback, initial_values, *args, **kwargs):
```

Comment:
We should look into whether we need this

Reply:
Agreed! Currently, I've copied and pasted another line to generate these pvs and it relies on the init statement `cav_pvs = {prefix: CavityPV(prefix, self.on_cavity_change, initial_values=init_vals[prefix],
                                         prefix=prefix) for prefix in init_vals.keys()}`

You made the watcher PV without any reliance on the init statement of self.on_cavity_change statement, so maybe I can get away with deleting some excess lines of code